src/LanguageServer.ts

```diff
@@ -1244,7 +1245,7 @@ export class LanguageServer {
             await Promise.all(this.getProjects().map(project => {
                 return project.builder.program.getReferences(srcPath, position);
             })),
-            c => c
+            c => c ?? []
```

Comment:
Existing implementation can return `undefined` instead of an array, which is filtered out below. As long as the `filter` line is present, returning `[]` vs `undefined` have equivalent behavior. I did not remove the `filter` because I don't know whether anything else in the array could be implicitly undefined.